purity/gc/gc.go

```diff
@@ -496,17 +496,20 @@ func (c *collector) RemoveActionResults() error {
 			ctx, cancel := context.WithTimeout(context.Background(), 1*time.Hour)
 			defer cancel()
 			for _, ar := range ars {
-				log.Debug("Removing action result %s", ar.Hash)
```

Comment:
Why are we getting rid of this debug log?

Reply:
We log above saying we have scheduled it for deletion so decided to remove this to try and reduce lines. the logfile produced by this is huge so thought it would be best to drop this one